proto/options.proto

```diff
@@ -303,12 +303,14 @@ message Costing {
     oneof has_restriction_probability {
       uint32 restriction_probability = 77;
     }
+
     repeated AvoidEdge exclude_edges = 78;
     oneof has_elevator_penalty {
       float elevator_penalty = 79;
     }
     uint32 fixed_speed = 80;
     uint32 axle_count = 81;
+    float use_lit = 82;
```

Comment:
looks like we have to do it a `oneof` here to satisfy the `JSON_PBF_RANGED_DEFAULT` macro @chrstnbwnkl , even though it wouldn't actually be necessary.

Reply:
ha, funny, reverting actually pulls up my initial comment again!